src/Mod/Path/Gui/Resources/panels/DlgJobTemplateExport.ui

```diff
@@ -112,7 +112,7 @@
    <item>
     <widget class="QGroupBox" name="settingsGroup">
      <property name="toolTip">
-      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enable to include the default values in the template.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enable to include values of the SetupSheet in the template.&lt;/p&gt;&lt;p&gt;Any values of the SetupSheet that are changed from their default are preselected. If this field not selected the current SetupSheet was not modified.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
@mlampert 
I think this should be `If this field _was_ not selected` according to the context...right?